tensorrt_llm/_torch/modules/kimi_kda/_kda_kernels.py

```
@@ -0,0 +1,390 @@
+        _PREFILL_IMPORT_ERROR = exc
+        raise
+    _PREFILL_MODULE = module
+    return module
+
+
+def is_intree_prefill_available() -> bool:
+    """True when the in-tree CuTe DSL prefill op can be imported."""
+    try:
+        _load_prefill_module()
+        return True
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)